### PR TITLE
Release/3.13.1

### PR DESCRIPTION
**Pull request by laurejt**

### CODE REVIEW

**Comment by laurejt** [comment]
@rlskoeser does the pucas update need to be included in the change log?

**Comment by rlskoeser** [comment]
> @rlskoeser does the pucas update need to be included in the change log?

Yes, probably worth a brief note - thanks for catching. Let me suggest a change

**Comment by laurejt** [comment]
> > @rlskoeser does the pucas update need to be included in the change log?
> 
> Yes, probably worth a brief note - thanks for catching. Let me suggest a change

Thank you, the suggested change will be much appreciated

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@
 CHANGELOG
 =========
 
+3.13.1
+----
+- Add support for Plausible analytics
+
```
Comment (rlskoeser) [inline CHANGELOG.rst:9]:
```suggestion
- Update admin login template to correctly include pucas styles and adjust spacing
```

**Pull request merged**